src/ip/QosConfig.h

```diff
@@ -43,12 +44,12 @@ class acl_nfmark
     CBDATA_CLASS(acl_nfmark);
 
 public:
-    acl_nfmark() : next(NULL), aclList(NULL), nfmark(0) {}
+    acl_nfmark() : next(NULL), aclList(NULL), markConfig(NfMarkConfig::Empty()) {}
```

Comment:
with NfConfig using C++11 initialization this construction becomes completely unnecessary. Just removing the nfmark mention is sufficient.

If you want to go all the way this class can be C++11 initialized as well and the custom constructor removed completely - or made to be "acl_nfmark() = default;"